.gitignore

```diff
@@ -1 +1,7 @@
 /target
+
+# Cargo
+Cargo.lock
```

Comment:
Hm I suppose this is fine. https://doc.rust-lang.org/cargo/guide/cargo-toml-vs-cargo-lock.html 
Can you do this in a separate pr?